Dockerfiles/Dockerfile

```diff
@@ -8,8 +8,9 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     curl -L -o /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH} \ 
     && chmod a+x /usr/local/bin/operator-sdk && \
     mkdir /project/output && \
-    dnf install --setopt=install_weak_deps=False -y git-core ansible && \
+    dnf install --setopt=install_weak_deps=False -y git-core ansible python3-pip mlocate && \
     dnf clean all && \
+    pip3 install operator-courier && \
     git clone --branch $OPERATOR_TEST_PLAYBOOKS_TAG https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
 COPY ./entrypoint.sh /project/entrypoint.sh
 ENTRYPOINT ["/project/entrypoint.sh"]
```

Comment:
Why do we change this file ?
operator-courier is unrelated to emty annotation change.

Reply:
In order to test an empty annotation file, it has to go through prepare-operator-metadata role. 
Which requires operator-courier
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/prepare-operator-metadata.yml
Which needs operator-courier to nest copy the metadata. 
I tested the role with the same playbook thus requiring prepare-operator-metadata.yml

Reply:
The same commend as in different thread. the Dockerfle for midstream must be used just for midstream and nothing else.